src/compas_xr/assemblies/assemblies.py

```diff
@@ -0,0 +1,94 @@
+
+import os
+from compas.data import json_dumps,json_loads
+from compas_xr.storage import Storage
+from compas_xr.realtime_database import RealtimeDatabase
+
+class AssemblyAssistant(object):
+
+    def __init__(self, config_path):
+
+        if os.path.exists(config_path):
+            self.storage = Storage(config_path)
+            self.database = RealtimeDatabase(config_path)
+
+        # Raise Exception that it could not be founda at that path.
+        else:
+            raise Exception("Could not create Storage or Database with path {}!".format(config_path))
```

Comment:
This is only a suggestion, but in cases in which the failure mode is clear and up-front, I usually prefer to invert the logic and raise first, so that the rest of the method is less indented:
```suggestion
        if not os.path.exists(config_path):
            raise Exception("Path not found. Could not create Storage or Database with path {}!".format(config_path))

        self.storage = Storage(config_path)
        self.database = RealtimeDatabase(config_path)
```

Reply:
Done :)

Reply:
Done :)